tool/src/test/java/org/wildfly/security/tool/MaskCommandTest.java

```diff
@@ -63,9 +64,9 @@ public void testMissingSaltAndIteration() {
         String retVal = executeCommandAndCheckStatusAndGetOutput(args);
         String[] retValLines = retVal.split(LINE_SEPARATOR);
 
-        assertTrue("Message about invalid salt parameter must be present", retValLines[0].contains("Invalid \"salt\" parameter. Generated value"));
-        assertTrue("Message about invalid iteration parameter must be present", ("Invalid \"iteration\" parameter. Default value \"" + defaultIteration + "\" will be used.").equals(retValLines[1]));
-        assertTrue("Message about invalid salt parameter must be present", retValLines[2].contains("MASK-"));
+        assertEquals("Message about invalid salt parameter must be present", true, retValLines[0].contains("Invalid \"salt\" parameter. Generated value"));
+        assertEquals("Message about invalid iteration parameter must be present", true, ("Invalid \"iteration\" parameter. Default value \"" + defaultIteration + "\" will be used.").equals(retValLines[1]));
```

Comment:
this statement is passing `true` as the expected value and then passes this line:`("Invalid \"iteration\" parameter. Default value \"" + defaultIteration + "\" will be used.").equals(retValLines[1])` as the actual value to the `assertEquals` function to confirm that they are equal. we can simplify this by just passing `("Invalid \"iteration\" parameter. Default value \"" + defaultIteration + "\" will be used.")` as the expected value and `retValLines[1]` as the actual value to the `assertEquals` function and the function will compare to check if the 2 strings are equal.

Reply:
okay, thanks. Fixing this.